packages/create-astro/src/index.ts

```diff
@@ -337,42 +337,40 @@ export async function main() {
 		console.log(`  You can safely ignore these files, but don't delete them!`);
 		console.log(dim('  (ex: tsconfig.json, src/env.d.ts)'));
 		console.log(``);
-		tsResponse.typescript = 'default';
+		tsResponse.typescript = 'base';
 		await wait(300);
 	}
 	if (args.dryRun) {
 		ora().info(dim(`--dry-run enabled, skipping.`));
 	} else if (tsResponse.typescript) {
-		if (tsResponse.typescript !== 'default') {
-			const templateTSConfigPath = path.join(cwd, 'tsconfig.json');
-			fs.readFile(templateTSConfigPath, (err, data) => {
-				if (err && err.code === 'ENOENT') {
-					// If the template doesn't have a tsconfig.json, let's add one instead
-					fs.writeFileSync(
-						templateTSConfigPath,
-						stringify({ extends: `astro/tsconfigs/${tsResponse.typescript}` }, null, 2)
-					);
+		const templateTSConfigPath = path.join(cwd, 'tsconfig.json');
+		fs.readFile(templateTSConfigPath, (err, data) => {
+			if (err && err.code === 'ENOENT') {
+				// If the template doesn't have a tsconfig.json, let's add one instead
+				fs.writeFileSync(
+					templateTSConfigPath,
+					stringify({ extends: `astro/tsconfigs/${tsResponse.typescript}` }, null, 2)
```

Comment:
wonder if it's helpful linking to the docs for this in a comment as well?
<https://docs.astro.build/en/guides/typescript/#setup>

Reply:
Not a bad idea! I didn't change those lines (just dedented them), so doesn't necessarily need to be added in this PR, but might as well toss it in, feel free to add a commit for that.